scss/detail.scss

```diff
@@ -0,0 +1,39 @@
+$gray : #828282;
+
+.book-container {
+  display: flex;
```

Comment:
No need to add this since you are already using `column` class in your HTML tag

Reply:
Done